content/en/docs/reference/_index.md

```diff
@@ -21,7 +21,7 @@ This section of the Kubernetes documentation contains references.
 * [Kubernetes API Overview](/docs/reference/using-api/api-overview/) - Overview of the API for Kubernetes.
 * Kubernetes API Versions
   * [1.17](/docs/reference/generated/kubernetes-api/v1.17/)
-  * [1.16](/docs/reference/generated/kubernetes-api/v1.16/)
+  * [1.16](https://v1-16.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/)
   * [1.15](/docs/reference/generated/kubernetes-api/v1.15/)
   * [1.14](/docs/reference/generated/kubernetes-api/v1.14/)
   * [1.13](/docs/reference/generated/kubernetes-api/v1.13/)
```

Comment:
Does it make sense to make the same change for v1.13 to v1.15?

(I'm not sure what's best)

Reply:
@sftim, Yes, I _think_ so. See my questions above ☝️
This would create changes during release time.

Reply:
I'm fine with either option so long as it makes sense to people.